helsingin-yliopisto-teologinen-tiedekunta-teologian-tyyli.csl

```diff
@@ -0,0 +1,478 @@
+      </else>
+    </choose>
+  </macro>
+  <macro name="issue">
+    <choose>
+      <if type="legal_case">
+        <text variable="authority" prefix=". "/>
+      </if>
+      <else-if type="speech">
+        <group prefix=" " delimiter=", ">
+          <text macro="event"/>
+          <text macro="day-month"/>
+          <text variable="event-place"/>
+        </group>
+      </else-if>
+      <else-if type="article-newspaper article-magazine" match="any">
+        <text macro="day-month"/>
+      </else-if>
+      <else>
+        <group prefix=" " delimiter=", ">
+          <choose>
+            <if type="thesis">
+              <text variable="genre" text-case="capitalize-first"/>
+            </if>
+          </choose>
+          <text macro="publisher" prefix=" "/>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="note">
+    <text variable="note" prefix="(" suffix=")"/>
+  </macro>
+  <citation et-al-min="4" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true" givenname-disambiguation-rule="primary-name">
+    <layout delimiter="; " suffix=".">
+      <choose>
+        <if match="none" variable="author editor translator">
+          <group delimiter=", ">
+            <group delimiter=" ">
+              <text macro="title" strip-periods="false"/>
+              <choose>
+                <if variable="issued">
+                  <date variable="issued">
+                    <date-part name="year"/>
+                  </date>
+                </if>
+                <else>
+                  <text term="no date" font-style="italic"/>
+                </else>
+              </choose>
+            </group>
+            <group>
+              <text variable="locator"/>
+            </group>
+          </group>
+        </if>
+        <else>
+          <group delimiter=", ">
+            <group delimiter=" ">
+              <text macro="contributors-short"/>
+              <choose>
+                <if match="any" variable="issued">
+                  <date date-parts="year" form="numeric" variable="issued"/>
+                </if>
+                <else>
+                  <text term="no date" font-style="italic"/>
+                </else>
+              </choose>
+            </group>
+            <group>
+              <text variable="locator"/>
+            </group>
+          </group>
+        </else>
+      </choose>
+    </layout>
+  </citation>
+  <bibliography hanging-indent="true" et-al-min="11" et-al-use-first="7" subsequent-author-substitute="" entry-spacing="2">
+    <sort>
+      <key macro="contributors"/>
+      <key macro="issued-year"/>
+      <key variable="title"/>
+    </sort>
+    <layout suffix=".">
+      <choose>
+        <if variable="author editor translator" match="none">
+          <group delimiter=". ">
+            <text macro="title"/>
+            <text macro="issued-year" display="left-margin" prefix="(" suffix=")"/>
```

Comment:
What's the goal of the `display` element here? It doesn't really make sense nor do I see anything unusual about formatting in the style guide.

Reply:
You're right, it's completely useless. I guess it was left over from the template style I used. I haven't done anything like this before, so the style overall probably isn't as neatly done as it could be 😀 I'll fix it!